plugins/modules/postgresql_publication.py

```diff
@@ -210,19 +226,11 @@
 from ansible.module_utils.basic import AnsibleModule
 from ansible.module_utils.six import iteritems
 from ansible_collections.community.postgresql.plugins.module_utils.database import (
-    check_input,
-    pg_quote_identifier,
-)
+    check_input, pg_quote_identifier)
 from ansible_collections.community.postgresql.plugins.module_utils.postgres import (
-    connect_to_db,
-    ensure_required_libs,
-    exec_sql,
-    get_conn_params,
-    get_server_version,
-    pg_cursor_args,
-    postgres_common_argument_spec,
-    set_comment,
-)
+    connect_to_db, ensure_required_libs, exec_sql, get_conn_params,
+    get_server_version, pg_cursor_args, postgres_common_argument_spec,
+    set_comment)
```

Comment:
```suggestion
    connect_to_db, ensure_required_libs, exec_sql, get_conn_params,
    get_server_version, pg_cursor_args, postgres_common_argument_spec,
    set_comment)
```
does it anyhow relate to the PR? could you please put it back. Even in places where I agree with refactoring, it should go to a separate PR. Let's not mix unrelated things in a single PR

Reply:
@Andersson007 I followed the contributing guide and this is a change applied by the pre-commit hook.

Reply:
oh, sorry, we should remove that thing from the guide - it does too much.. thanks for following the guide